From a7a2fd28dc4180f1cbadf6038b67337ae6fdee34 Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Thu, 26 May 2016 14:01:35 +0200
Subject: [PATCH] fw_env.c: include stdint.h unconditionally

later builds fail with
| tools/env/fw_env.c:52:2: error: unknown type name 'uint8_t'
|   uint8_t mtd_type;  /* type of the MTD device */

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 tools/env/fw_env.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/env/fw_env.c b/tools/env/fw_env.c
index 1173eea..edc5090 100644
--- a/tools/env/fw_env.c
+++ b/tools/env/fw_env.c
@@ -15,6 +15,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <stddef.h>
+#include <stdint.h>
 #include <string.h>
 #include <sys/types.h>
 #include <sys/ioctl.h>
@@ -22,7 +23,6 @@
 #include <unistd.h>
 
 #ifdef MTD_OLD
-# include <stdint.h>
 # include <linux/mtd/mtd.h>
 #else
 # define  __user	/* nothing */
-- 
2.6.6

